### ISSUE TITLE
race condition when enableLazyRendering and target="some-input"

### ISSUE THREAD

**Issue by a15n** [sir-dunxalot/ember-tooltips#150]
The tooltip will appear then immediately disappear when the input is clicked on...

```
<input id="some-input">

{{tooltip-on-element
    target="#some-input"
    text="hello world"
    event="click"
    enableLazyRendering=true
}}
```

I believe this has something to do with focus events occurring before click events... will investigate

![race condition](https://cloud.githubusercontent.com/assets/7050871/21558171/75a17f0a-cdea-11e6-89f2-31958d5256bc.gif)

PS: I saw the same thing for popover-on-element

**Comment by a15n** [comment]
@jiongbozhang  FYI

**Comment by a15n** [comment]
it seems like we can get around this if we use `event='focus'` (as is shown in the dummy app). Should we try to make a race condition fix for this or just encourage people to use focus?

**Comment by sir-dunxalot** [comment]
We should fix the race condition because this is a common use case for tooltips in Ember.

**Comment by sir-dunxalot** [comment]
#149 is now on master.

**Comment by a15n** [comment]
PR here https://github.com/sir-dunxalot/ember-tooltips/pull/153

**Comment by sir-dunxalot** [comment]
Fixed by `2.7.3`.
